### DISCUSSION TITLE
Como puedo obtener la data de el cliente en el flujo principal

### DISCUSSION THREAD

**Discussion by AlvinTLC** [codigoencasa/builderbot#522] [General]
Quiero llamar al api de mis clientes que se ejecute un if

if ctx.from == from.db algo asi solo lo explico no quiero entrar en contexto

Si el numero de telefono que escribio hola, coincide con uno de la base de datos, flow 2 y responder, Buenas Estimado Fulano de Tal
Su cuenta tiene un balance pendiente de DatosFromAPI.balancependiente

que mientras este cliente este en la conversacion y la conversacion no sea marcada como cerrada osea la sessio de chat entonses quede con las variables

si el cliente no esta en el api que salte el flow 3, donde le mostrare los planes y servicios y de mas.

**Comment by leifermendez** [comment]
@AlvinTLC creo que con los floyDynamic lo puedes lograr puedes ver un video aqui https://bot-whatsapp.netlify.app/docs/masterclass/
